tests/Aggregate/AggregateTranslatorConfigurationTest.php

```diff
@@ -0,0 +1,327 @@
+<?php
+/**
+ * This file is part of the prooph/service-bus.
+ * (c) 2014-2016 prooph software GmbH <contact@prooph.de>
+ * (c) 2015-2016 Sascha-Oliver Prolic <saschaprolic@googlemail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ProophTest\EventStore\Aggregate;
+
+use PHPUnit_Framework_TestCase as TestCase;
```

Comment:
I copied an existing testcase and it was implemented like that. Furthermore PHPUnit\Framework\TestCase is not available since PHPUnit 4.8 is used.